pkg/bosh/manifest/manifest.go

```diff
@@ -511,17 +509,16 @@ func (m *Manifest) ApplyAddons(ctx context.Context) error {
 			continue
 		}
 		for _, ig := range m.InstanceGroups {
-			include, err := m.addOnPlacementMatch(ctx, "inclusion", ig, addon.Include)
+			include, err := m.addOnPlacementMatch("inclusion", ig, addon.Include)
 			if err != nil {
 				return errors.Wrap(err, "failed to process include placement matches")
 			}
-			exclude, err := m.addOnPlacementMatch(ctx, "exclusion", ig, addon.Exclude)
+			exclude, err := m.addOnPlacementMatch("exclusion", ig, addon.Exclude)
 			if err != nil {
 				return errors.Wrap(err, "failed to process exclude placement matches")
 			}
 
 			if exclude || !include {
-				ctxlog.Debugf(ctx, "Addon '%s' doesn't match instance group '%s'", addon.Name, ig.Name)
```

Comment:
This was really useful when kubecf was starting.
Addon match rules are complicated, and we don't have any other feedback for understanding why an addon is or isn't applied.

Reply:
Hm, maybe we need another solution here. Maybe just passing a zap logger and logging only "interesting" cases. The addons were several thousand lines of logging.